### PR TITLE
fluttips

### PR DESCRIPTION
**Pull request by mtnbarreto**
added blogpost

### CODE REVIEW

File: _posts/2023-02-09-Fluttips.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+permalink: /blog/fluttips/
+---
+
+<img width="100%" src="/images/fluttips/Blog_Fluttips.png)" />
+
+### Introduction
+
+At Xmartlabs, we believe in empowering young talent and providing them with training and professional instruction. We are committed to their technical development, which is a top priority for us. This is why we've created internal projects like Fluttips, a mobile app that streamlines the Flutter development experience.
+
+Fluttips collects tips and tricks from expert sources, such as Vandadnp's GitHub libraries and Flutter & Google video tips, into a one-stop shop for Flutter developers. Users will be able to browse through a vast amount of knowledge and find the information they need with no need to scan several sites.
```
Comment (mirland) [inline _posts/2023-02-09-Fluttips.md:18]:
```suggestion
Fluttips collects tips and tricks from expert sources, such as [Vandadnp's GitHub][tips-images-repo] libraries and [Flutter & Google video tips][tips-videos-repo], into a one-stop shop for Flutter developers. Users will be able to browse through a vast amount of knowledge and find the information they need with no need to scan several sites. 
```

Add the references at the bottom

File: _posts/2023-02-09-Fluttips.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+category: development
+featured_image: /images/fluttips/Blog_Fluttips.png
+permalink: /blog/fluttips/
+---
+
+<img width="100%" src="/images/fluttips/Blog_Fluttips.png)" />
+
+### Introduction
+
+At Xmartlabs, we believe in empowering young talent and providing them with training and professional instruction. We are committed to their technical development, which is a top priority for us. This is why we've created internal projects like Fluttips, a mobile app that streamlines the Flutter development experience.
```
Comment (mirland) [inline _posts/2023-02-09-Fluttips.md:16]: We need a link in Fluttips

File: _posts/2023-02-09-Fluttips.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Collaborate with the Community: Contributing to the Open-Source Fluttips Repo
+
+If you are interested in contributing to Fluttips, you can access our open-source repo on GitHub via the link: [https://github.com/xmartlabs/fluttips](https://github.com/xmartlabs/fluttips). Make any necessary contributions and add value to the app. We are excited to collaborate with the community to make Fluttips an even better tool!
+
+### Conclusion
+
+Fluttips is a powerful tool for Flutter developers that offers a wealth of knowledge on the framework's functionalities and features. It's an excellent resource for developers looking to implement specific functionalities or discover new characteristics of Flutter, as well as for those who are new to the framework. Fluttips is designed with a user-friendly interface and its open-source nature encourages collaboration and knowledge-sharing among developers.
+
+Fluttips provides an efficient and enjoyable experience for all users and provides an excellent opportunity to contribute to an open-source project. Make the most of this powerful tool and join us in making Fluttips an even better resource!
```
Comment (mirland) [inline _posts/2023-02-09-Fluttips.md:72]: Don't forget to add the links here: tips-images-repo and tips-videos-repo

Comment (mtnbarreto) [reply _posts/2023-02-09-Fluttips.md:72]: @mirland  o sol podrian agregarlos? suggest en github sin clonar

Comment (mirland) [reply _posts/2023-02-09-Fluttips.md:72]: Yep

File: _posts/2023-02-09-Fluttips.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Explore the Benefits of Fluttips - A Comprehensive Flutter Resource for Developers
+date: '2023-02-09T09:00:00.000-03:00'
```
Comment (mirland) [inline _posts/2023-02-09-Fluttips.md:4]:
```suggestion
date: '2023-02-10T09:00:00.000-03:00'
```

**Review by mirland — APPROVED** [review]

**Pull request merged**